libs/common/src/test-helpers/mock-data/record-correction-log-details.mock.ts

```diff
@@ -0,0 +1,23 @@
+export const recordCorrectionLogDetailsMock = {
```

Comment:
type please for better easier maintenance

Reply:
done